Plugin/Magento/Framework/App/AreaList.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\UpwardConnector\Plugin\Magento\Framework\App;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class AreaList
+{
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * Controller or frontname to load from default magento frontend
+     */
+    public const UPWARD_CONFIG_PATH_FRONT_NAMES_TO_SKIP = 'web/upward/front_names_to_skip';
```

Comment:
Constants are public by default, we don't need to explicitly define that

Reply:
Fixed by d8bde4b